person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = id
```

Comment:
- [ ] you're creating an instance variable `@id` passing an argument of `id` however this is not an argument that is passed, what you can do in this case is to assign a random number to this instance variable.

Reply:
Thank you, I will assign random number to instance variable id.